src/Extensions/NexusMods.Extensions.Hashing/AbsolutePathExtensions.cs

```diff
@@ -39,6 +39,8 @@ public static class AbsolutePathExtensions
     /// <returns>The xxHash64 hash of the file.</returns>
     public static Hash XxHash64MemoryMapped(this AbsolutePath input, IActivitySource<Size>? job = null)
     {
+        if (input.FileSystem is InMemoryFileSystem) 
+            return XxHash64MemoryMappedInMemory(input, job);
```

Comment:
This shouldn't be needed. The `InMemoryFileSystem` supports memory mapped files. The entire point of having a memory mapped file API in `IFileSystem` was so that consumers **don't** have to check for the FS implementation.

@Sewer56 wrote that API after my nagging about this.

Reply:
I never actually got that code to work, because of how half-implemented our in memory filesystem is. I'd love to execute tests completely in memory, but the InMemoryFilesystem has a lot of edge cases currently.

Reply:
removed